libraries/engage/back-in-stock/components/Subscriptions/components/Subscription/index.jsx

```diff
@@ -0,0 +1,181 @@
+  PriceInfo,
+  CrossIcon,
+} from '@shopgate/engage/components';
+import { getProductRoute, ProductImage } from '@shopgate/engage/product';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { themeConfig } from '@shopgate/engage';
+import PriceStriked from '@shopgate/pwa-ui-shared/PriceStriked';
+import Price from '@shopgate/pwa-ui-shared/Price';
+import AvailableText from '@shopgate/pwa-ui-shared/Availability';
+import { BackInStockButton } from '@shopgate/engage/back-in-stock/';
+import {
+  getThemeSettings, i18n,
+} from '@shopgate/engage/core';
+import { useBackInStockSubscriptions } from '@shopgate/engage/back-in-stock/hooks';
+
+const { variables } = themeConfig;
+
+const styles = {
+  root: css({
+    display: 'flex',
+    position: 'relative',
+    marginBottom: 16,
+  }).toString(),
+  imageContainer: css({
+    flex: 0.4,
+    maxWidth: 170,
+    minWidth: 70,
+  }).toString(),
+  infoContainer: css({
+    flex: 1,
+  }).toString(),
+  infoContainerRow: css({
+    display: 'flex',
+    justifyContent: 'space-between',
+  }).toString(),
+  baseContainerRow: css({
+    flexDirection: 'column',
+    display: 'flex',
+    marginTop: '8px',
+  }).toString(),
+  priceContainerRow: css({
+    flexDirection: 'column',
+    display: 'flex',
+    alignItems: 'end',
+    marginTop: '8px',
+  }).toString(),
+  priceContainer: css({
+    minWidth: 100,
+  }).toString(),
+  priceInfo: css({
+    wordBreak: 'break-word',
+    fontSize: '0.875rem',
+    lineHeight: '0.875rem',
+    color: 'var(--color-text-low-emphasis)',
+    padding: `${variables.gap.xsmall}px 0`,
+    textAlign: 'right',
+  }).toString(),
+  titleContainer: css({
+  }).toString(),
+  title: css({
+    fontSize: 17,
+    fontWeight: 600,
+    flexWrap: 'wrap',
+    overflowWrap: 'anywhere',
+  }).toString(),
+  removeContainer: css({
+    minWidth: '30px',
+  }).toString(),
+  availabilityText: css({
+    fontSize: '0.875rem',
+  }).toString(),
+};
+
+/**
+ * Renders on single Back in Stock subscription
+ * @param {Object} props Props.
+ * @param {string} props.subscription The subscription which should be rendered
+ * @returns {JSX}
+ */
+const Subscription = ({
+  subscription,
+}) => {
+  const { subscriptionCode, product } = subscription;
+  const {
+    removeBackInStockSubscription,
+  } = useBackInStockSubscriptions();
+  const { ListImage: gridResolutions } = getThemeSettings('AppImages') || {};
+  const currency = product.price?.currency || 'EUR';
+  const defaultPrice = product.price?.unitPrice || 0;
+  const specialPrice = product.price?.unitPriceStriked;
+  const hasStrikePrice = product.price?.discount > 0;
+  const productLink = getProductRoute(product.id);
+
+  return (
+    <div className={styles.root}>
+      <Link
+        className={styles.imageContainer}
+        component="div"
+        href={productLink}
+        aria-hidden
+      >
+        <ProductImage src={product.featuredImageBaseUrl} resolutions={gridResolutions} />
```

Comment:
please add some spacing around the product image. right now there isn't any gap between the image and the content block.